include/spdlog/sinks/minute_file_sink.h

```diff
@@ -0,0 +1,217 @@
+// Copyright(c) 2015-present, Gabi Melman & spdlog contributors.
```

Comment:
Please remove this file as we use the other one

Reply:
done